### PR TITLE
User can delete

### PR DESCRIPTION
**Pull request by danieljonesdmj**
users can edit and delete posts

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -11,7 +15,25 @@ def create
...
+    if @post.update_attributes(post_params)
+      redirect_to posts_path, :notice => 'Your post has been updated'
+    end
+
+  end
+
+  def destroy
+    @post = Post.find(params[:id])
+    @post.destroy
+    redirect_to posts_path
```
Comment (bibicollins) [inline app/controllers/posts_controller.rb]: Add a notice?

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -16,3 +16,37 @@
...
+/* #edit {
+  background-color: #4CAF50;
+}
+
+#delete {
+  background-color: #f44336;
+} */
+
+#text {
+  color: white !important;
```
Comment (bibicollins) [inline app/assets/stylesheets/application.css]: what is !important?

Comment (danieljonesdmj) [reply app/assets/stylesheets/application.css]: the !important overrides all other css. We were trying to make our edit and delete buttons not look like hyperlinks but haven't figured it out yet.

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (danieljonesdmj) [reply app/controllers/posts_controller.rb]: Added a notice

**Pull request merged**